CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ set(WITH_XC_ALL OFF CACHE BOOL "Build in all available plugins")
 option(WITH_XC_AUTOTYPE "Include Auto-Type." ON)
 option(WITH_XC_NETWORKING "Include networking code (e.g. for downloading website icons)." OFF)
 option(WITH_XC_BROWSER "Include browser integration with keepassxc-browser." OFF)
+option(WITH_XC_BROWSER_PASSKEYS "Passkeys support for browser integration." OFF)
```

Comment:
I am not sure we need to make this a feature flag, seems benign to any user whether it is included or not (ie, no new dependencies and no security implications to include it or not). It is also entirely under the umbrella of the browser extension which has its own feature flag. You would still need an internal flag though due to the Qt 5.12 requirement. Although this entirely goes away if we actually move 2.8.0 to Qt 6.

Reply:
We still need a flag for now, but shouldn't be a feature flag (ie, set on the cmake command line)